tests/catboost/test_catboost_model_export.py

```diff
@@ -419,3 +420,15 @@ def test_pyfunc_serve_and_score_sklearn(reg_model):
     )
     scores = pd.read_json(resp.content.decode("utf-8"), orient="records").values.squeeze()
     np.testing.assert_array_almost_equal(scores, model.predict(inference_dataframe.head(3)))
+
+
+def test_log_model_with_code_paths(cb_model):
+    artifact_path = "model"
+    with mlflow.start_run(), mock.patch(
+        "mlflow.catboost._add_code_from_conf_to_system_path"
```

Comment:
Is there a way to test that `_add_code_from_conf_to_system_path` works? Can we introduce this test coverage?

Reply:
I believe the pytorch tests with the module_scoped_subclassed_model will test that adding code to system paths work since the logged code needs to be added to the system path to serve the model

Reply:
Seems pretty indirect. Can we test this directly?

Reply:
Yep, just added a test in test_model_utils.py!